src/protocol.h

```diff
@@ -107,7 +107,7 @@ extern void send_key_changed(void);
 extern bool send_req_key(struct node_t *to);
 extern bool send_ans_key(struct node_t *to);
 extern bool send_tcppacket(struct connection_t *c, const struct vpn_packet_t *packet);
-extern bool send_sptps_tcppacket(struct connection_t *c, const char *packet, int len);
+extern bool send_sptps_tcppacket(struct connection_t *c, const char *packet, size_t len);
```

Comment:
Maybe the `const char *` should become a `const void *` or `const uint8_t *` as well?